### PR TITLE
Add method to payout request allocation

### PR DESCRIPTION
**Pull request by colinbird**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by dara123 — APPROVED** [review]

**Review by i-yasko — APPROVED** [review]

File: openapi/components/schemas/PayoutRequest.yaml [hunk 1]
```diff
@@ -84,6 +84,8 @@ properties:
             - canceled
             - declined
             - unknown
+        method:
```
Comment (ashkarpetin) [inline openapi/components/schemas/PayoutRequest.yaml:87]: Should it be called paymentMethod?

Comment (colinbird) [reply openapi/components/schemas/PayoutRequest.yaml:87]: I see we use both, but we use `method` more often, including in paymentinstrument

**Review by lexerom — APPROVED** [review]

File: openapi/components/schemas/PayoutRequest.yaml (hunk 1 above)
Comment (ashkarpetin) [reply openapi/components/schemas/PayoutRequest.yaml:87]: We use it in context of paymentInstrument: paymentInstrument.method.

Comment (colinbird) [reply openapi/components/schemas/PayoutRequest.yaml:87]: Seems like a wider question, we also use method inside token.  I am not sure

**Review by ashkarpetin — APPROVED** [review]

**Review by nanobash — APPROVED** [review]

**Pull request merged**